packages/client/components/GitLabScopingSearchResults.tsx

```diff
@@ -97,6 +98,7 @@ const GitLabScopingSearchResults = (props: Props) => {
                       edges {
                         node {
                           ... on _xGitLabProject {
+                            fullPath
```

Comment:
`repoIntegration` requires `fullPath` to determine whether it's a GitLab project. See rootSchema

Reply:
this is a pretty big issue, `allProjects` is actually resolving to `projects`. I try to explain why in the loom. It's 90 minutes, the summary starts at 1:25:00.

https://www.loom.com/share/79e4d705572c40bdba449b12bc3250b7

Reply:
i baked this functionality into nest-graphql-endpoint in v0.6.0, i'll submit a PR later to bump versions so we never git bitten by this again